AppDashboard/templates/layouts/panel.html

```diff
@@ -0,0 +1,26 @@
+<div class="panel panel-default" id="panel{{ panel_count }}">
```

Comment:
I just realized I forgot to do this one, however I'm having difficulty having that and having panels being added (since those won't have the correct id) unless I make the dictionary have another item for the nav area that has the id that it can use to look up in the look up dictionary... thoughts?